recipes/cad1/README.md

```diff
@@ -0,0 +1,91 @@
+# The 1st Cadenza Challenge
+Cadenza challenge code for the 1st challenge.
+
+For more information about the Cadenza Challenge please visit https://cadenzachallenge.com
+
+Cadenza tutorials are now available in https://cadenzachallenge.com.
```

Comment:
Is there a specific page for the tutorials or is it the [learning resources](http://cadenzachallenge.org/docs/learning_resources/learning_intro)? If its the learning resources perhaps the following...

```suggestion
[Learning Resources](http://cadenzachallenge.org/docs/learning_resources/learning_intro) are available on the challenge homepage.```

Reply:
I missed that the URL here is incorrect too, it should be `.org` and not `.com` even if links to the specific tutorials aren't available for inclusion in this PR.